test/Fields/field.jl

```diff
@@ -283,13 +283,14 @@ end
 end
 
 # Test truncated field type printing:
-ClimaCore.Fields.truncate_printing_field_types() = true
+import Preferences
+Preferences.@set_preferences!(Pair("TruncateClimaCoreFieldPrinting" => true))
```

Comment:
This probably won't work, as it needs to be set before the module is loaded.

Reply:
Yeah, I wasn't sure but I thought I'd try. What's a good way to test this, then? Can I add a separate `@safetestset "Recursive" begin @time include("test_truncated_field_printing.jl") end`?

Reply:
Let me try that

Reply:
no, it has to be in a whole new session.

Reply:
(it will also trigger precompilation)